include/openPMD/Iteration.hpp

```diff
@@ -99,8 +136,33 @@ class Iteration : public Attributable
     void flush();
     void read();
 
+    enum class CloseStatus
+    {
+        Open,
+        ClosedInFrontend,
+        ClosedInBackend
+    };
+
+    /*
+     * An iteration may be logically closed in the frontend,
+     * but not necessarily yet in the backend.
+     * Will be propagated to the backend upon next flush.
+     * Store the current status.
+     */
+    std::shared_ptr< CloseStatus > m_closed =
+        std::make_shared< CloseStatus >( CloseStatus::Open );
+
+    /**
+     * @brief Verify that a closed iteration has not been wrongly accessed.
```

Comment:
"wrongly accessed" is too vague, better be concrete what is not allowed:
```suggestion
     * @brief Verify that a closed iteration has not been accessed and/or manipulated further.
```